__tests__/components/TableRow.spec.ts

```diff
@@ -0,0 +1,370 @@
+            [mockMosaicValue, mockNamespaceRowValue].forEach((rowValue) => {
+                runBasicHasMetadataTests(
+                    {
+                        ...rowValue,
+                        metadataList: [],
+                    },
+                    false,
+                );
+            });
+        });
+    });
+
+    describe('aliasActionLabel', () => {
+        const runBasicAliasActionLabel = (params, expectedResult) => {
+            // Arrange:
+            const wrapper = getTableRowWrapper({
+                rowValues: {
+                    ...params,
+                },
+            });
+
+            const vm = wrapper.vm as TableRowTs;
+
+            // Act + Assert:
+            // @ts-ignore
+            expect(vm.aliasActionLabel).toBe(expectedResult);
+        };
+
+        test('returns alias link when namespace value alias type is N/A', () => {
+            runBasicAliasActionLabel(
+                {
+                    ...mockNamespaceRowValue,
+                    aliasType: 'N/A',
+                },
+                'action_label_alias_link',
+            );
+        });
+
+        test('returns alias unlink when namespace value alias type is mosaic', () => {
+            runBasicAliasActionLabel(
+                {
+                    ...mockNamespaceRowValue,
+                    aliasType: 'mosaic',
+                },
+                'action_label_alias_unlink',
+            );
+        });
+
+        test('returns alias link when row value is mosaic and not link with namespace', () => {
+            runBasicAliasActionLabel(
+                {
+                    ...mockMosaicValue,
+                    name: 'N/A',
+                },
+                'action_label_alias_link',
+            );
+        });
+
+        test('returns alias unlink when row value is mosaic and linked with namespace', () => {
+            runBasicAliasActionLabel(
+                {
+                    ...mockMosaicValue,
+                    name: mockNamespaceRowValue.name,
+                },
+                'action_label_alias_unlink',
+            );
+        });
+    });
+
+    describe('visibleRowValues', () => {
+        const runBasicVisibleRowValues = (params, expectedResult) => {
+            // Arrange:
+            const hiddenData = {
+                value: 'value',
+            };
+
+            const wrapper = getTableRowWrapper({
+                rowValues: {
+                    ...params,
+                    hiddenData,
+                },
+            });
+
+            const vm = wrapper.vm as TableRowTs;
+
+            // Act + Assert:
+            // @ts-ignore
+            expect(vm.visibleRowValues).toEqual(params);
+            // @ts-ignore
+            expect(vm.visibleRowValues).not.toHaveProperty('hiddenData');
+            expect(wrapper.findComponent(AmountDisplay).exists()).toBe(expectedResult);
```

Comment:
imo, this is separate test; AmountDisplay is only present for mosaic but not namespace

Reply:
yeah, that makes sense, it's not "unit" enough if it has multiple assertions.